mlir/test/Conversion/ArithToEmitC/arith-to-emitc-cast-unsupported.mlir

```diff
@@ -0,0 +1,48 @@
+// RUN: mlir-opt -split-input-file --pass-pipeline="builtin.module(convert-arith-to-emitc{float-to-int-truncates})" -verify-diagnostics %s
+
```

Comment:
Maybe you can not set `float-to-int-truncates` here and show the errors that occur then? (could also be a different test, I have no strong opinion on that)